docs/src/pages/index.mdx

```diff
@@ -1,7 +1,71 @@
-import { Callout } from 'nextra/components'
- 
-<Callout emoji="🏗️">
- We are building our new website, click [here to vist the docs](/docs)
-</Callout
+---
+title: GrapqhQL Debugger
+searchable: false
+---
```

Comment:
```suggestion
```

Lets remove this for the time being as we need to do a round of SEO on all pages.